exchange/src/shared/config/config_loader.hpp

```diff
@@ -49,7 +49,8 @@ class Config {
     static const Config&
     get_instance()
     {
-        static Config instance(CONFIG_FILE);
+        const char* env_var = std::getenv("CONFIG_FILE");
+	static Config instance((env_var!=nullptr)? std::string(env_var):CONFIG_FILE);
```

Comment:
```suggestion
        const char* config_file_path = std::getenv("NUTC_CONFIG_FILE");
        if (!config_file_path)
            config_file_path = CONFIG_FILE;

	static Config instance(config_file_path);
```

also i'd rename the `CONFIG_FILE` macro to `DEFAULT_CONFIG_FILE`.